presidio-analyzer/analyzer/predefined_recognizers/credit_card_recognizer.py

```diff
@@ -33,6 +33,10 @@ def __init__(self):
     def validate_result(self, pattern_text, pattern_result):
         sanitized_value = CreditCardRecognizer.__sanitize_value(pattern_text)
         res = CreditCardRecognizer.__luhn_checksum(sanitized_value)
+
+        # Send interpretability traces
+        pattern_result.append_analysis_explenation_text(
+            "Credit card checksum result: {}".format(str(res)))
```

Comment:
Why not add a field for checksum/verification logic? This is quite generic. The textual representation should be used only for specific use cases

Reply:
Do you mean that  all recognizers will have checksum/verification logic?
I also thought the point was to make an example that utilizes the solution

Reply:
The generic analysis_explanation should contain a value for checksum/validation too. 
We can provide other examples if necessary, but the validation is a part of the generic process and could be automatically traced too.